maestro-app/src/main/resources/file_centric.json

```diff
@@ -79,25 +79,52 @@
           "file_autocomplete"
         ]
       },
+      "file_type": {
+        "type": "keyword"
+      },
+      "file_access": {
+        "type": "keyword"
+      },
       "analysis" : {
         "properties" : {
-          "id" : {
+          "analysis_id" : {
             "type": "keyword",
             "copy_to": [
               "file_autocomplete"
             ]
           },
+          "analysis_type": {
+            "type": "keyword"
+          },
+          "analysis_version": {
+            "type": "integer"
+          },
+          "workflow": {
```

Comment:
this should be removed I didn't see it before, it's part of the dynamic mappings

Reply:
it's in the updated file centric mapping: https://github.com/icgc-argo/argo-metadata-schemas/blob/master/mappings/maestro_file_centric.json#L86